modules/localization/msf/common/util/compression.cc

```diff
@@ -107,6 +107,14 @@ int ZlibStrategy::ZlibUncompress(BufferStr* src, BufferStr* dst) {
   stream_data.opaque = Z_NULL;
   stream_data.avail_in = 0;
   stream_data.next_in = Z_NULL;
+  stream_data.total_in = 0ULL;
```

Comment:
Please don't try to init every member in struct one by one, which was hard to read/maintain.  Use `std::memset(...)` from <cstring> to zero-initialize the whole struct first.

Reply:
ok